pkg/apis/aws/v1alpha1/accountclaim_types.go

```diff
@@ -52,9 +53,28 @@ type AccountClaimCondition struct {
 	Message string `json:"message,omitempty"`
 }
 
-// AccountClaimConditionType  is a valid value for AccountClaimCondition.Type
+// AccountClaimConditionType is a valid value for AccountClaimCondition.Type
 type AccountClaimConditionType string
 
+const (
+	// AccountClaimed is set when an Account is claimed
+	AccountClaimed AccountClaimConditionType = "Claimed"
+	// AccountUnclaimed is set when an Account is not claimed
+	AccountUnclaimed AccountClaimConditionType = "Unclaimed"
+)
+
+// ClaimStatus is a valid value from AccountClaim.Status
+type ClaimStatus string
```

Comment:
@nimrodshn See the ClaimStatus type you should be watching a claim for status.state=Ready

Reply:
@jharrington22 Amazing thanks!